### PR TITLE
Update AerServ Adapter for Prebid 1.0

### PR DESCRIPTION
**Pull request by envision753**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
AerServ Adapter for Prebid 1.0

- test parameters for validating bids
```
//banner:
{
  bidder: 'aerserv',
  params: {
    plc: 1028582
  }
}

//video:
{
  bidder: 'aerserv',
  params: {
    plc: 1028581,
    video: {}
  }
}
```

- contact email of the adapter’s maintainer: technology@aerserv.com
- [x] official adapter submission

### CODE REVIEW

File: modules/aerservBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    const bidResponses = [];
+
+    if (typeof response !== 'object' || response.error) {
+      return bidResponses;
+    }
+
+    if (isValidResponse(response, request.bidRequest)) {
+      const bidResponse = {
+        requestId: request.bidRequest.bidId,
+        bidderCode: spec.code,
```
Comment (dbemiller) [inline modules/aerservBidAdapter.js]: You shouldn't set this. Adapter aliases won't work quite right if you do. We'll set it automatically in the `bidderFactory`.

Our docs are being updated now... sorry for the confusion.

**Comment by dbemiller** [comment]
One more change, now that #1742 got merged.

The first argument to interpretResponse now looks like this:

```
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

You'll have to update the spec so that it looks through that object as well now.

**Comment by mkendall07** [comment]
@envision753 
Any updates on this?

**Comment by mkendall07** [comment]
@envision753 
another ping.

**Comment by mkendall07** [comment]
closing due to inactivity.

**Pull request closed without merging**